test_pull_requests

```diff
@@ -1006,11 +1103,37 @@ popd
             status = result == 'SUCCESS' ? 'success' : 'failure'
           end
         else
+          # in this case, we're in one of three states:
```

Comment:
You are inconsistent with capitalization.  This starts with a lower case,  but the below paragraph starts with upper.